### DISCUSSION TITLE
In usefieldArray if one  checkbox is checked all other checkbox field should be false

### DISCUSSION THREAD

**Discussion by NarenderSiri** [react-hook-form/react-hook-form#8546] [Q&A]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by bluebill1049** [comment]
as `useFieldArray` it's best to wrap your controlled component with checkboxes and have a state (checked value) hoist inside that controlled component.

```tsx
const Checkbox = () => {
  // useController
  // useState
  return <input />
}

{fields.map((field) => {
  return <div key={field.id}>
    <Checkbox />
  </div>
})}
```

**Reply by NarenderSiri** [reply]
Sir,how can i use in useFormContext

**Reply by bluebill1049** [reply]
It's the same usage except you get methods from the context instead of props pass down from parent components.

**Reply by NarenderSiri** [reply]
please check the CodeSandbox      [CodeSandBox](https://codesandbox.io/s/reacttableandreacthookformscheckbox-v1n301?file=/src/Columns.js)  .
In a group of multiple checkbox, I want to check only single checkbox and not to be checked other but its checking multiple checkbox..
It is not working as expected.
Please help me.

https://user-images.githubusercontent.com/48355455/175241505-57a3135a-d753-42dc-a631-dd04092a7696.mp4
